### PR TITLE
Add favicons and touch icons

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -60,5 +60,5 @@ gulp.task('scss-unused-variables', getTask('scss-unused-variables'));
 gulp.task('ci', ['scss-lint', 'scss-unused-variables']);
 
 gulp.task('build', function (done) {
-    runSequence('clean:dist', 'sass:build', 'svgs-generate', 'jekyll:docs', 'docs:copy-components', 'fingerprint', 'fingerprint-replace', 'sass:docs-build', done);
+    runSequence('clean:dist', 'sass:build', 'sass:docs-build', 'svgs-generate', 'jekyll:docs', 'docs:copy-components', 'fingerprint', 'fingerprint-replace', done);
```
Comment (aju) [inline gulpfile.js]: We need to fingerprint docs css as well so it needs to be done before fingerpinting tasks

File: scripts/tasks/fingerprint-replace.js [hunk 2]
```diff
@@ -3,9 +3,11 @@ module.exports = function (gulp, plugins, consts) {
         var manifest = require(plugins.path.join(consts.VERSIONED_DIST, 'rev-manifest'));
         var cssPath = plugins.path.join(consts.VERSIONED_DIST, 'style-guide.css');
         var docsHtmlPath = plugins.path.join(consts.VERSIONED_DIST, 'docs', '*.html');
+        var docsCssPath = plugins.path.join(consts.VERSIONED_DIST, 'docs', 'css', 'main.css');
```
Comment (aju) [inline scripts/tasks/fingerprint-replace.js]: Fingerprinting of docs css file

File: scripts/tasks/fingerprint.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-        var images = plugins.path.join(consts.SRC, 'images', '*');
-        var logos = plugins.path.join(consts.SRC, 'images', 'logos', '*');
+        var images = plugins.path.join(consts.SRC, 'images', '**', '*');
+        var icons = plugins.path.join(consts.SRC, 'images', 'icons', '*');
+        var subjects = plugins.path.join(consts.SRC, 'images', 'subjects', '*');
 
         // by default, gulp would pick `assets/css` as the base,
         // so we need to set it explicitly:
-        return gulp.src([fonts, images, logos], {base: './src'})
+        return gulp.src([fonts, images, '!' + icons, '!' + subjects], {base: './src'})
```
Comment (aju) [inline scripts/tasks/fingerprint.js]: We exclude from fingerprinting just icons and subject images as they are build into the sprite and not used as single units

**Comment by mr-mig** [comment]
:+

**Comment by matzimowski** [comment]
:+

**Pull request merged**